### ISSUE TITLE
Article: [Improving your website accessibility]

### ISSUE THREAD

**Issue by Emilsone** [a11yproject/a11yproject.com#1333]
Delete this boilerplate or any other information that doesn't apply to your article.

Please review our [Code of Conduct](/code-of-conduct/), [Contributor documentation](https://a11yproject.com/contributing-guidelines/), and [Content Style Guide](https://a11yproject.com//content-style-guide/) before submitting content.

## Description

>A brief description (2-3 sentences) of your proposed article. Why should this article be written, and how does it fit on The A11Y Project?

Most websites are not accessible because web accessibility is overlooked by developers, designers, content creators Because they feel most people don't use the web in a traditional way, But in this article, I'm going to take you through what web accessibility is? benefits of web accessibility and how to make your website accessible.

## Would you like to write the article or request that it be written?
I would like to write the article

>Are you posting this as a means to look for help on the topic or do you have a method you've developed/tested to solve the issue/topic?
No, I don't need help on the topic, I have a method to solve it

**Comment by ericwbailey** [comment]
Hi @Emilsone, thank you for pitching this! The topic you've provided is a little broad. Do you have an outline in mind for what you'd specifically like to cover?

**Comment by Emilsone** [comment]
Hi @ericwbailey, thanks for the response. Below is a link to my draft that contains the outline 
https://www.dropbox.com/scl/fi/ohw2y6zqr8ga4w605qati/Untitled.paper?dl=0&rlkey=qi4o6eu9mrtcdf7ntbc6ix512

**Comment by Emilsone** [comment]
Hello @ericwbailey, please can you approve the article?

**Comment by ericwbailey** [comment]
@Emilsone Mind sharing the outline in a format we can review it in? A Google Doc or Dropbox paper link work, or just a GitHub comment here.

**Comment by Emilsone** [comment]
Hi @ericwbailey  below is a link to my outline draft 

https://www.dropbox.com/scl/fi/ohw2y6zqr8ga4w605qati/Improving-your-website-accessibility.paper?dl=0&rlkey=qi4o6eu9mrtcdf7ntbc6ix512

**Comment by ericwbailey** [comment]
@Emilsone I'd like to see more clarity around the "How to make your website accessible?" bullet point. It doesn't have to be a ton, just enough to let me know what you intend to cover. I say this because this is a huge topic, well beyond the scope of a single blog post.

**Comment by Emilsone** [comment]
> @Emilsone I'd like to see more clarity around the "How to make your website accessible?" bullet point. It doesn't have to be a ton, just enough to let me know what you intend to cover. I say this because this is a huge topic, well beyond the scope of a single blog post.

Hi, @ericwbailey Please kindly check the document and leave your feedback over there. Thanks

**Comment by ericwbailey** [comment]
@Emilsone For transparency, I'm replying here. I'm not seeing any clarification around the "How to make your website accessible?" bullet point, as requested. Closing this issue for now, I might suggest posting this on a platform such as [dev.to](https://dev.to/) instead.
